.gitignore

```diff
@@ -11,6 +11,7 @@ ui_*.h
 *.pro.user*
 qrc_*
 docs/html
+.directory
```

Comment:
A normally invisible file that the file manager on kubuntu used to track
files in the directory.
On Jan 10, 2015 1:40 PM, "Thorbjørn Lindeijer" <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/bjorn/tiled/pull/859#discussion-diff-22762158>:
>
> > @@ -11,6 +11,7 @@ ui_*.h
> >  *.pro.user*
> >  qrc_*
> >  docs/html
> > +.directory
>
> What is the .directory?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bjorn/tiled/pull/859/files#r22762158>.
>